timemachine/fe/rbfe.py

```diff
@@ -793,10 +787,10 @@ def run_edge_and_save_results(
     pkl_obj = (mol_a, mol_b, edge.metadata, core, solvent_res, solvent_top, complex_res, complex_top)
     file_client.store(path, pickle.dumps(pkl_obj))
 
-    solvent_ddg = np.sum(solvent_res.result.all_dGs)
-    solvent_ddg_err = np.linalg.norm(solvent_res.result.all_errs)
-    complex_ddg = np.sum(complex_res.result.all_dGs)
-    complex_ddg_err = np.linalg.norm(complex_res.result.all_errs)
+    solvent_ddg = sum(r.dG for r in solvent_res.final_result.bar_results)
```

Comment:
Could `PairBarResult` add properties to return all_dGs, all_errs, etc to make it simpler to read the values by lambda?

Reply:
Added in https://github.com/proteneer/timemachine/pull/974/commits/f4a479e837d37c1b41a1d1c7efbac88809c4824f